lib/features/composer/presentation/composer_view.dart

```diff
@@ -490,6 +496,57 @@ class ComposerView extends GetWidget<ComposerController> with AppLoaderMixin {
     );
   }
 
+  Widget _buildRowIconStyleText() {
+    return Row(
+      children: [
+        _buildIconStyleText(
+            path: RichTextStyleType.bold.getIcon(imagePaths),
+            isSelected: controller.richTextMobileTabletController.isTextStyleTypeSelected(RichTextStyleType.bold),
+            onTap: () async {
+             await controller.htmlEditorApi?.formatBold().then((value) {
+               controller.richTextMobileTabletController.selectTextStyleType(RichTextStyleType.bold);
+             });
+            }
+        ),
+        _buildIconStyleText(
+            path: RichTextStyleType.italic.getIcon(imagePaths),
+            isSelected: controller.richTextMobileTabletController.isTextStyleTypeSelected(RichTextStyleType.italic),
+            onTap: () async {
+              await controller.htmlEditorApi?.formatItalic().then((value) {
+                controller.richTextMobileTabletController.selectTextStyleType(RichTextStyleType.italic);
+              });
+            }
+        ),
+        _buildIconStyleText(
+            path: RichTextStyleType.underline.getIcon(imagePaths),
+            isSelected: controller.richTextMobileTabletController.isTextStyleTypeSelected(RichTextStyleType.underline),
+            onTap: () async {
+              await controller.htmlEditorApi?.formatUnderline().then((value) {
+                controller.richTextMobileTabletController.selectTextStyleType(RichTextStyleType.underline);
+              });
+            }
+        ),
+      ],
+    );
+  }
+
+  Widget _buildIconStyleText({
+    required String path,
+    required bool? isSelected,
+    required VoidCallback onTap,
+  }){
+    return buildIconWeb(
+        icon: SvgPicture.asset(
+            path,
+            color: isSelected == true ? AppColor.colorNameEmail : AppColor.colorDividerMailbox,
```

Comment:
Should not do that, please define the dedicated color for your widgets. What happens in the future if you change the color of `NameEmail`? You will be hard to manage the color of other widgets like that.

In this case, enable/disable color for style button can be the same.